File: C_Sharp_code/Example/WindowsFormsApp/CS_demo21/Antelope.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace CS_demo21
{
    public class Antelope : Animal
    {
        public string name;

        public Antelope()
        {
        }

        public Antelope(string name)
        {
            this.name = name;
        }

        public override void sleep()
        {
            Console.WriteLine("羚羊在睡覺");
        }

        public override string ToString()
        {
            return "這是羚羊";
        }
    }
}

File: C_Sharp_code/Example/WindowsFormsApp/VB_demo22/Antelope.vb
Public Class Antelope
    Inherits Animal
    Public name As String
    Public Sub New()

    End Sub
    Public Sub New(ByVal argName As String)
        Me.name = argName
    End Sub

    Public Overrides Function ToString() As String
        Return "這是羚羊"
    End Function

    Public Overrides Sub Sleep()
        Console.WriteLine("羚羊在睡覺")
    End Sub
End Class
